llmfoundry/utils/builders.py

```diff
@@ -196,6 +198,16 @@ def build_algorithm(name: str, kwargs: Dict[str, Any]) -> Algorithm:
                                    kwargs=kwargs)
 
 
+def build_metric(name: str, kwargs: Dict[str, Any]) -> Metric:
```

Comment:
can we make this kwargs optional or be **kwargs? Don't like that we need to pass in `{}` everytime

Reply:
I didn't want to do **, cause then you can't have kwargs to your constructor that match other args in the function, but it can be optional, sure